release_bot/github.py

```diff
@@ -412,18 +387,16 @@ def pr_exists(self, name):
         :param name: name of the PR
         :return: PR number if exists, False if not
         """
-        cursor = ''
-        while True:
-            edges = self.walk_through_prs(start=cursor, direction='before', closed=False)
-            if not edges:
-                self.logger.debug(f"No open PR's found")
-                return False
-
-            for edge in reversed(edges):
-                cursor = edge['cursor']
-                match = re.match(name, edge['node']['title'].lower())
-                if match:
-                    return edge['node']['number']
+        merged_prs = self.walk_through_prs(PRStatus.open)
```

Comment:
I'm a bit confused here. Are you looking for merged or opened PRs?

```suggestion
        opened_prs = self.walk_through_prs(PRStatus.open)
```